src/crypto/crypto-ops.c

```diff
@@ -1512,7 +1512,7 @@ static void ge_precomp_cmov(ge_precomp *t, const ge_precomp *u, unsigned char b)
   fe_cmov(t->xy2d, u->xy2d, b);
 }
 
-static void select(ge_precomp *t, int pos, signed char b) {
+static void select_pos(ge_precomp *t, int pos, signed char b) {
```

Comment:
I'm curious about the potential error here, for the reasons stated above (i.e. cotire is now another hidden target).